src/services/CollectionsApi.ts

```diff
@@ -0,0 +1,17 @@
+import { CollectionFlow, CollectionFlowResponse } from '@/models'
+import { Api, ApiRoute } from '@/services/Api'
+import { mapper } from '@/services/Mapper'
+import { mocker } from '@/services/Mocker'
+
+export class CollectionsApi extends Api {
+
+  protected override route: ApiRoute = '/collections'
+
+  public getFlowCollection(): Promise<CollectionFlow[]> {
+    return Promise.resolve(mocker.createMany('collectionFlow', 10))
+    // return this.get<CollectionFlowResponse[]>('views/flow')
```

Comment:
Does this need to be removed?

Reply:
currently there is no api for this, but yes when [this](https://github.com/PrefectHQ/nebula/pull/3542) gets merged, we will want to swap this out